seqr/utils/elasticsearch/es_search.py

```diff
@@ -322,7 +324,7 @@ def _filter_compound_hets(self, quality_filters_by_family, annotations_secondary
             for pair_index, families in paired_index_families[index].items():
                 paired_families.update({family: pair_index for family in families})
 
-            for family_guid in sorted(family_samples_by_id.keys()):
+            for family_guid in family_samples_by_id.keys():
```

Comment:
why would you take away sorting?

Reply:
I think `sorted()` is not necessary because the code in the loop is not depending on the order.

Reply:
After removing the sorting on the filters, the sorted() here becomes necessary. Added it back.
Ideally, we shouldn't add `sorted()` here because it doesn't change the search function but wastes some computation.